sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 - Fixed an issue where `BearerTokenAuthenticationPolicy` throws `ArgumentOutOfRangeException` if the `ExpiresOn` property of the token is the default value. ([#47040](https://github.com/Azure/azure-sdk-for-net/pull/47040))
+- Fixed an issue where `ManagedIdentityCredential` does not honor the `CancellationToken` passed to `GetToken` and `GetTokenAsync`. ([#47156](https://github.com/Azure/azure-sdk-for-net/issues/47156))
```

Comment:
Curious: if the fix is in the Identity library, why add the comment to the Azure.Core changelog?

Reply:
That would be because I had the wrong editor window open at the time 🤦‍♂️

Reply:
You would not be the first person to have done such a thing! :)